bin/docserv-createconfig

```diff
@@ -229,6 +229,13 @@ def get_git_branches(path):
             result[branch_wo_origin] = []
     return result
 
+def get_args(args):
+    if not args.languages:
+        args.languages = input("# Add languages: ")
+    if not args.filter:
+        args.filter = input("# Filter: ")
+    if not args.contact:
+        args.contact = input("# Contact: ")
```

Comment:
As discussed, when the user hits return, the following will happen:
- `args.langs` is set to `en_us`, since **all** our Docs are available in english.
- `args.filter` is left empty, because we don't always need this.
- `args.contact` is the tricky one, throw an error maybe? Minimum one email address needed here.